.storybook/preview-head.html

```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="preview.css">
```

Comment:
I think you should import global/style.css file instead of creating a new one for storybook.

Reply:
If we do this, then we will need to include font inside the body style of website which affect the whole website. Also other styles present in the `global.css` will affect the storybook rendering. I think we should keep it seperate.

Reply:
We are only using the `preview.css` for updating storybook font as it can't be done in the storybook configuration file. The components rendered in the storybook are the same as in the in components folder and we don't need to write any extra component or style for using it in storybook.

Reply:
Then why we aren't able to see Tailwind CSS properties using classes in the storybook.

Reply:
Got your point, I have resolved it. Now we won't need  `preview-head.html` and `preview.css` files to include the font in storybook. I have removed them.